docs/content/stable/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -123,24 +137,66 @@ You can make a standby instance active as follows:
 
 You should be able to see that all of the data has been restored into the instance, including universes, users, metrics, alerts, task history, cloud providers, and so on.
 
-## Check results
+### Verify failover or switchover
+
+After switching or failing over to the standby, verify that the old active universe is in standby mode (switchover) or no longer available (failover). If both YBA instances attempt to take actions on DB universes, it could cause unpredictable side effects.
+
+**Switchover**
+
+- [Verify that HA is functioning properly](#verify-ha).
+- If the old active universe is not in standby mode, there could be an issue with communication from the new active to the old active universe. Follow the [setup instructions](#configure-active-and-standby-instances) to verify that certificates and ports are set up correctly.
+
+**Failover**
+
+- If the old active universe is hard down, verify that there is no chance that it can come back and run YBA at a later point.
+- If the old active universe does come back up, it should automatically go into standby mode. If it does not go into standby mode, you should manually demote it as follows:
+
+    1. Find the correct last failover time by querying the new active instance.
+
+        ```sh
+        curl -X GET 'https://10.9.104.164/api/v1/settings/ha/config' -H 'X-AUTH-YW-API-TOKEN: 8bf6e5b2-beff-4859-9016-bdc0fa0ced1e' --insecure
+        ```
+
+    1. Send a manual demote request to the stale active.
+
+        ```sh
+        curl -X PUT 'https://10.9.113.130/api/v1/settings/ha/internal/config/demote/1705138628028' -H 'X-AUTH-YW-API-TOKEN: 1a5ad6de-f638-4ebc-9649-d8b187b688b0' -H 'HA-AUTH-TOKEN: sTroAGbJz+QydXXz9lc1bhdQmQqIZSyM6Z20MqitvLA=' -H "Content-Type: application/json" --data-raw '{"leader_address": "https://10.9.104.164"}' --insecure
```

Comment:
We should get rid of any internal information in these requests. I would think that means IP and API tokens. instead of 104.164 and 113.130 we should use <STALE_ACTIVE> <TRUE_ACTIVE> or something that communicates similar meaning. Also, the timestamp (1705138628028) is the response given by the first API call. I'm not sure if we want this in public docs at all though, this feels like more a KB article that support can reference and guide the customer through. We probably don't want people randomly trying to demote actives it's only if they get into a weird state, which ideally should never happen. cc @iSignal

Reply:
@iSignal keep or drop?

Reply:
Discussed with Sanketh, we should drop this and link to the https://github.com/yugabyte/yugabyte-db/blob/master/managed/api-examples/python-simple/high-availability.ipynb "Manual HA Demotion". I have added the steps there.